resources/js/Pages/Dashboard.jsx

```diff
@@ -0,0 +1,222 @@
+import AuthenticatedLayout from "@/Layouts/AuthenticatedLayout";
+import { Head } from "@inertiajs/react";
+import axios, * as others from "axios";
+import { useState, useRef } from "react";
+import Card from "react-bootstrap/Card";
+import { useMemo, useEffect } from "react";
+import { debounce } from "lodash";
+import { Navbar } from "react-bootstrap";
+import "./../../css/custom.css";
+// import { Container } from "postcss";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+
+const Dashboard = ({ auth, accessToken }) => {
+    const [isLoad, setIsLoad] = useState(false);
+    const [isUniSelected, setIsUniSelected] = useState(false);
+    const [noDataFound, setNoDataFound] = useState(false);
+
+    const timer = null;
```

Comment:
where is this used?

Reply:
I was initailly using timeout hook for debounce but then I used lodash debounce so just forgot to remove this line.